File: app/src/main/java/com/georgcantor/wallpaperapp/ui/util/WallpViewHolder.java
package com.georgcantor.wallpaperapp.ui.util;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import com.georgcantor.wallpaperapp.R;

public class WallpViewHolder extends RecyclerView.ViewHolder {

    public DynamicHeightImageView discWallp;

    public WallpViewHolder(View itemView) {
        super(itemView);
        discWallp = itemView.findViewById(R.id.wallpView);
    }
}

File: app/src/main/java/com/georgcantor/wallpaperapp/ui/util/WallpViewHolder.kt
package com.georgcantor.wallpaperapp.ui.util

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import kotlinx.android.synthetic.main.wallp_item.view.*

class WallpViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var imageView: DynamicHeightImageView = itemView.wallpView
}
